app/src/main/res/values/strings.xml

```diff
@@ -82,4 +81,9 @@
     <string name="title_bill_demo">250.000$</string>
     <string name="msg_error_form">Please enter full information in the form at the side</string>
     <string name="msg_error_phone">Please enter the phone number</string>
+    <string name="title_space_time">Space time</string>
+    <string name="title_end_date">End date</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_end_date` appears to be unused
 Unused resources make applications larger and slow down builds.